### ISSUE TITLE
"The git reference could not be found"

### ISSUE THREAD

**Issue by jeremiahlee** [SunderJS/sunder#5]
I am unable to install Sunder. I have tried on 2 different MacBook Pros (M1 and Intel), using npm v8.3.0 and 8.1.0 (respectively), and node v17.3.0 and v16.12.0.

```shell
% npm i --save sunder    
npm ERR! code 1
npm ERR! The git reference could not be found
npm ERR! command git --no-replace-objects checkout 2.0
npm ERR! error: pathspec '2.0' did not match any file(s) known to git
```

**Comment by vicary** [comment]
Using `pnpm` reveals a bit more about the underlying error,

```
$ pnpm i sunder
ERROR  Could not resolve 2.0 to a commit of git://github.com/jshttp/http-errors.git.
```

Should be related to the released version, digging deeper shows that this is in fact fixed at master by 802cd23582641e8adde84056f624d2543fccc930.

A patch release as a hot fix should happen ASAP.

**Comment by vicary** [comment]
Current workaround, install the GitHub version.

```
$ npm i sunder@github:SunderJS/sunder
```

Directly installing HEAD should be discouraged, at least pinning down the commit is recommeded.

```
$ npm i sunder@github:SunderJS/sunder#802cd23582641e8adde84056f624d2543fccc930
```

**Comment by gzuidhof** [comment]
I just released 0.10.1 which includes the NPM version of the http-errors package.
